.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": ["react", "es2015", "stage-0"],
```

Comment:
> It's mean that I only get what I need?

Yes, let's what I mean. Since you don't know what you want to use now. Do you think it's a good idea to add when you want it?